backends/platform/libretro/lite_engines.list

```diff
@@ -0,0 +1,32 @@
+agi
```

Comment:
How is this going to be maintained? Are you going to watch new engines being added and try to keep this list in sync?

Reply:
This list is coming from the legacy core, and it's currently used for wiiu builds only. I suppose there are size limitations on that platform, though I don't know the full detail of the matter.
The list will have no need to be maintained (unless currently listed engines do not change names), as it will not be expanded.